### DISCUSSION TITLE
Working on PR Linter

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1166] [Ideas] [answered]
@open-sauced/maintainers @bdougie 
I've been working on getting that PR linter implemented.

https://github.com/mtfoley/pr-compliance-action

I'd love to get some input on what you think of these!
criteria | effect | notes
--- | --- | ---
PR not referring to issue | leave comment, close PR | prob needs to exclude dependabot PRs
PR head is protected branch (e.g. main) | leave comment, close PR |
PR title not following convention | leave comment |
PR modifies certain files (e.g. package.json) | leave comment, mention maintainers |

**Comment by mtfoley** [comment]
So yeah, I'm almost done with this thing, just lacks a proper readme and I need to figure out how to release. 🚀

**Reply by 0-vortex** [reply]
I'll be available for you today whenever you wanna do this, ping a bit in advance if you wanna pair!

**Reply by mtfoley** [reply]
I think I could prob just use a repo other than one of mine to try it on.

**Comment by mtfoley** [comment, marked as answer]
I set it up for a pre-release. Curious how it does in practice 😃 
https://github.com/marketplace/actions/pr-compliance-action

just realized I forgot to put a sample workflow file in the docs!
